core/src/network/event.rs

```diff
@@ -109,7 +109,7 @@ where
     /// A connection may close if
     ///
     ///   * it encounters an error, which includes the connection being
-    ///     closed by the remote. In this case `error` is `Some`.
+    ///     closed by the remote. In this case `error` is `ome`.
```

Comment:
```suggestion
    ///     closed by the remote. In this case `error` is `Some`.
```

Reply:
Fixed in https://github.com/libp2p/rust-libp2p/pull/2218. Unfortunately I can not apply suggestions on a branch on the comit-network repository.